src/server/utils/validation.ts

```diff
@@ -26,11 +26,7 @@ export function isCountryNameValid(countryName: string): boolean {
   return countriesList.some((country) => country.value === countryName);
 }
 
-export function throwIfConfigVarIsUndefined(varName: string): void {
```

Comment:
I don't know why this exists tbh, I think all the code defaults to `?? ""` so they'll never be undefined.. 😭

Reply:
should I leave it or get rid?

Reply:
leave for now.. just airing a grievance 😮‍💨